test-support/trie-bench/CHANGELOG.md

```diff
@@ -5,29 +5,34 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.28.0] - 2021-07-02
+- Updated `memory-db to 0.27.
```

Comment:
It's worth noting here as well that parity-scale codec requirement has been loosened up to `^2` again